docs/section-2/_category_.json

```diff
@@ -1,4 +1,4 @@
 {
-  "label": "TLIP Infrustructure",
+  "label": "TLIP Infrustracture",

```

Comment:
Still wrong 😄 should be `Infrastructure`

Reply:
The second commit fixes it